Tasks/UseDotNetV2/Tests/L0.ts

```diff
@@ -452,3 +452,39 @@ describe('UseDotNet', function () {
         }, tr, done);
     });
 });
+
+describe('globaljsonfetcher', function () {
```

Comment:
It'll be better if a new section is not described. As this goes as a heading in the logs. It is usually used to print the task name for which tests are written so that they can easily be grouped while browsing through logs.
You can place the tests in the already present `describe` section `UseDotNet`